chef/cookbooks/cinder/templates/default/cinder.conf.erb

```diff
@@ -2559,14 +2559,12 @@ connection=<%= @sql_connection %>
 # Minimum number of SQL connections to keep open in a pool.
 # (integer value)
 # Deprecated group/name - [DEFAULT]/sql_min_pool_size
-# Deprecated group/name - [DATABASE]/sql_min_pool_size
```

Comment:
Note that you can keep this comment (and the ones that are removed a few lines below), as it's in the upstream config file.